experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -305,20 +305,28 @@ function get_atmos_config_dict(coupler_dict::Dict, job_id::String)
     atmos_config_file = coupler_dict["atmos_config_file"]
     atmos_config_repo = coupler_dict["atmos_config_repo"]
     # override default or specified configs with coupler arguments, and set the correct atmos config_file
+    comms_ctx = ClimaComms.context()
+    @info atmos_config_repo
     if atmos_config_repo == "ClimaCoupler"
         @assert !isnothing(atmos_config_file) "Must specify `atmos_config_file` within ClimaCoupler."
-        @info "Using Atmos configuration from ClimaCoupler in $atmos_config_file"
+        if ClimaComms.iamroot(comms_ctx)
```

Comment:
it would be nice to have a wrapper for this - maybe a function `info_on_root` that takes in a context and a message, and does this check and calls `@info` internally